Circuit/Components/VacuumTubes/Triode.cs

```diff
@@ -142,6 +147,12 @@ public override void Analyze(Analysis Mna)
                     ik = Call.If(exk > -50, -G * Binary.Power(Ln1Exp(exk) / C, Gamma), 0);
                     //ig = Mna.AddUnknownEqualTo("i" + Name + "g", ig);
                     //ik = Mna.AddUnknownEqualTo("i" + Name + "k", ik);
+                    if (SimulateCapacitances)
+                    {
+                        Capacitor.Analyze(Mna, Name + "_cpg", p, g, 2.4e-12);
```

Comment:
Answer for both of your questions is 'yes' :) I've added parameters for them. But something seems to be wrong with integration when I add all 3 capacitances - not all of them are getting integrated here:
https://github.com/dsharlet/LiveSPICE/blob/0d59564ea4ab6dc64c0e7e9d16dadb2d050e4ae4/Circuit/Simulation/TransientSolution.cs#L123-L130
as they seem to be codependent?? Could you take a look? I've added sample circuit: 
https://github.com/Federerer/LiveSPICE/blob/9a929965b194e37f065c2dd8e5fb9dce41dcb708/Tests/Unstable/Triode%20stage.schx

Reply:
I'll take a look. I don't see any issues that could explain this in your code. As a quick question, does the same thing work if you build the capacitors manually in a schematic? 

A silly possibility: you could try using 'decimal' values for the capacitances instead of doubles. The reason this might matter is that the CAS converts doubles and decimals to (exact) rationals, and maybe the representation of these values as decimals would produce a much less problematic rational value?

You should actually change this regardless. See e.g. https://github.com/dsharlet/LiveSPICE/blob/master/Circuit/Components/Capacitor.cs#L15 for example.

Reply:
(I doubt that will actually fix the issue though. The only problem using double instead of rational should cause is to make circuit analysis/solving slower due to bigger arbitrary precision integer rationals.)

Reply:
Tried with capacitors added to schematic and the issue is still there. Changing to decimal doesn't help either.